# ScreenExpression.msg
uint8 EYE_AWAKE      = 0
uint8 EYE_SLEEP      = 1
uint8 EYE_SLEEPY     = 2
uint8 EYE_GRIN       = 3
uint8 EYE_SAD        = 4
uint8 EYE_LOOK_RANDOM = 5
uint8 ANIMATE_LOVE   = 6  # ❤️ Animatie
uint8 ANIMATE_STAR   = 7  # 🌟 Animatie

uint8 action  # De expressie die moet worden weergegeven op het scherm



